src/measure_theory/measure_space.lean

```diff
@@ -1735,6 +1750,77 @@ end
 
 end measure_sub
 
+
+lemma restrict_sub_eq_restrict_sub_restrict {Ω : Type*} [measurable_space Ω]
+  (μ ν : measure Ω) {s : set Ω} (h_meas_s : is_measurable s) :
+  (μ - ν).restrict s = (μ.restrict s) - (ν.restrict s) :=
```

Comment:
In the proof of this lemma, please only use `simp` to close a goal (this is to ensure that the proof remains robust when the simp-set changes). See https://leanprover-community.github.io/extras/simp.html#when-it-is-unadvisable-to-use-simp.
Try to either to use `simp` later so that it can close the goal, or use `squeeze_simp` (`simp only` is ok in the middle of a proof).